ConnectionException must be notified through the sourceCallback (#141)
mulesoft/docs-mule-sdk: modules/ROOT/pages/reconnection.adoc

[[starting-source]]
==== Starting Source

> A source is in the starting source stage when the onStart() method executes. If

Reviewer: ```suggestion
A source is in the starting source stage when the `onStart()` method executes. 
```

> a connection error exists, it must be communicated through the SourceCallback

Reviewer: ```suggestion
Connection errors must be communicated through the `SourceCallback`
```

> using the onConnectionException() method. The source will stop and attempt to

Reviewer: ```suggestion
using the `onConnectionException()` method. The source stops and attempts to
```

  try {
    connection.startStreaming();
  } catch(Exception e){

>     throw new ConnectionException(e, connection); //<1>

Reviewer 2: We should also modify the text that is above, which states that the exception must be thrown just like in operations.
